tests/dialects/test_dialect.py

```diff
@@ -1097,32 +1097,51 @@ def test_order_by(self):
 
     def test_json(self):
         self.validate_all(
-            "JSON_EXTRACT(x, 'y')",
+            "JSON_EXTRACT(x, '$.y')",
             read={
-                "mysql": "JSON_EXTRACT(x, 'y')",
+                "bigquery": "JSON_EXTRACT(x, '$.y')",
+                "duckdb": "x -> 'y'",
+                "doris": "x -> '$.y'",
+                "mysql": "JSON_EXTRACT(x, '$.y')",
                 "postgres": "x->'y'",
-                "presto": "JSON_EXTRACT(x, 'y')",
-                "starrocks": "x -> 'y'",
-                "doris": "x -> 'y'",
+                "presto": "JSON_EXTRACT(x, '$.y')",
+                "sqlite": "x -> '$.y'",
```

Comment:
is this correct?

Reply:
which one?

Reply:
sqlite, does it really take a true json path? or is it like postgres and just takes in 'y'

Reply:
they do, most of the stuff I wrote, I tested in the corresponding engines and double-checked with docs

Reply:
check out https://www.sqlite.org/json1.html#jex